SplunkPy - Fixed a Mirror In issue (XSUP-43248) (#38796)
demisto/content: Packs/SplunkPy/ReleaseNotes/3_2_3.md

#### Integrations

##### SplunkPy

> - Fixed an issue where the *Mirror In* mechanism has stopped working once an error was raised during the run of the ***get-modified-remote-data*** command.

Reviewer: ```suggestion
- Fixed an issue where the *Mirror In* mechanism has stopped when ***get-modified-remote-data*** command failed on error.
```
